include/rocksdb/statistics.h

```diff
@@ -374,6 +374,15 @@ enum Tickers : uint32_t {
   // # of files deleted immediately by sst file manger through delete scheduler.
   FILES_DELETED_IMMEDIATELY,
 
+  // The counters for error handler, not that, bg_io_error is the subset of
+  // bg_error and bg_retryable_io_error is the subset of bg_io_error
+  ERROR_HANDLER_BG_ERROR_COUNT,
+  ERROR_HANDLER_BG_IO_ERROR_COUNT,
+  ERROR_HANDLER_BG_RETRYABLE_IO_ERROR_COUNT,
+  ERROR_HANDLER_AUTORESUME_COUNT,
+  ERROR_HANDLER_AUTORESUME_RETRY_TOTAL_COUNT,
+  ERROR_HANDLER_AUTORESUME_SUCCESS_COUNT,
+
```

Comment:
I am not arguing against these statistics, but are they at all meaningful to the RocksDB user?  Do they know anything about the error handler and its potential states and failures?  Is this opening a can of worms to document?

Reply:
So those are the statistics requested by the internal users. And I believe all the users will benefit from the statistics since error handler is the default logic in RocksDB to deal with the write IO errors.